src/components/YourCompany/NameTranslations/AddNameTranslation.vue

```diff
@@ -0,0 +1,147 @@
+<template>
+  <div>
+    <confirm-dialog
+      ref="confirmTranslationDialog"
```

Comment:
Please add an id to the top-level div for this component, and add an "attach" property to this dialog. This will attach the dialog to this component instead of the app level, ie its code will be in this component's DOM tree, which allows you to test the dialog (since elements at the app level are not available when you mount the component).